content/applications/hr/lunch/products.rst

```diff
@@ -0,0 +1,59 @@
+========
+Products
+========
+
+Odoo’s *Lunch* app does not come with any products pre-configured. The individual products being
+offered must first be configured before orders can be placed.
+
+First, navigate to :menuselection:`Lunch app --> Configuration: Products`. Next, click the
```

Comment:
```suggestion
To add and configure products for the *Lunch* app, navigate to :menuselection:`Lunch app --> Configuration --> Products`. Next, click the
```
just for some added specificity 

Also, when it comes to using ":" in menus, that is specifically for options that appear in a sub-menu. So, if there was a sub-menu section like: Settings app --> Technical --> Discuss: Messages ... that works.
![Screenshot 2024-06-20 131927](https://github.com/odoo/documentation/assets/73958186/8846fb53-5d8d-4c83-9e87-5f8ef1f62e30)

But, when its just a step-by-step direction, that ":" is not needed.